### DISCUSSION TITLE
New Docker Installation

### DISCUSSION THREAD

**Discussion by TreyThurlow** [firefly-iii/firefly-iii#5239] [Firefly III]
Hello,

This is my first time trying to install Firefly III and I tried using the docker-compose file as well as the full cli method. Once it gets up and I put in the IP:port into the web browser it says: "The Firefly image has moved. For the latest version of Firefly III, use fireflyiii/core:latest" which I was using.
 
Has anyone heard of this issue and have a fix?
Any information would be greatly appreciated!

**Comment by JC5** [comment]
Make sure you have no fireflyiii or jc5x/firefly-iii images on your drive, that may be the issue.

**Reply by TreyThurlow** [reply]
Oh that makes sense. I will have to find where they get pulled to and see if if fixes it. 

I will update when I can and appreciate the help!

**Reply by TreyThurlow** [reply]
> Make sure you have no fireflyiii or jc5x/firefly-iii images on your drive, that may be the issue.

I made sure that the only firefly image that is on the drive is fireflyiii/core:latest which is what the documentation says to use, however I am getting the same error

**Reply by JC5** [reply]
Sorry, trying this locally returns a Firefly III instance as expected. I'm not really sure what the next step could be.

**Reply by TreyThurlow** [reply]
> 
> 
> Sorry, trying this locally returns a Firefly III instance as expected. I'm not really sure what the next step could be.

I was able to get it and found what may have been the issue. I had MySQL already installed and configured, which I believe is what was causing the conflict. I think that the database wasn't purged from the old firefly image. I uninstalled MySQL and it worked.
